.github/workflows/publish-to-maven.yaml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publishToMaven:
+    environment:
+      name: "MavenCentral"
+
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: 17
+
+      - name: Upload library
+        run: ./gradlew :TopsortAnalytics:publishAndReleaseToMavenCentral
```

Comment:
Is it possible to split up build and upload? I'd rather not mix the two.

Reply:
Split off :compileReleaseKotlin which is the build step that takes the longest (and it's where problems usually show up)